tests/stp/recovery/recoveryreplayperiod.csv

```diff
@@ -1,21 +1,21 @@
 action,ms,bytes,lang,code,repeat,minver,comment
 before,0,0,q,startproc["stprepperiod"],1,,"Start STP"
 before,0,0,q,.servers.startup[],1,,"Start connection management"
-before,0,0,q,.proc.sys "sleep 1",1,,"Wait for procs to start up"
+before,0,0,q,.proc.sys "sleep 10",1,,"Wait for procs to start up"
 before,0,0,q,stpHandle:gethandle[`stprepperiod],1,,"Open handle to STP"
 before,0,0,q,stpHandle(`.stplg.init;testlogdb),1,,"Create test stplog test directory"
 before,0,0,q,logdir:1_string stpHandle(`.stplg.dldir),1,,"Get location of log directory"
 before,0,0,q,startproc["rdball"],1,,"Start RDB"
-before,0,0,q,.proc.sys "sleep 1",1,,"Wait for procs to start up"
+before,0,0,q,.proc.sys "sleep 10",1,,"Wait for procs to start up"
 before,0,0,q,rdbHandle:gethandle[`rdball],1,,"Open handle to RDB"
 run,0,0,q,c1:rdbHandle"count trade",1,,"Get initial trade table count"
 run,0,0,q,stpHandle(`.u.upd;`trade;testtrade),1,,"Send trade update to segmented tickerplant"
-run,0,0,q,.proc.sys "sleep 11",1,,"Wait for 10 second period to elapse"
+run,0,0,q,.proc.sys "sleep 15",1,,"Wait for 10 second period to elapse"
```

Comment:
comment is wrong (it already was wrong, but it's even more wrong now)

Reply:
rectified